myapp/forms.py

```diff
@@ -0,0 +1,85 @@
+from django import forms
+from django.contrib.auth import authenticate
+from django.contrib.auth.models import User
+from django.core.validators import MinValueValidator, MaxValueValidator
+
+
+class CheckRequirementsForm(forms.Form):
+    SEX_CHOICES = (
+        ('m', 'male'),
+        ('f', 'female')
+    )
+
+    CEFR = (
+        ("A1", "Beginner"),
+        ("A2", "Elementary"),
+        ("B1", "Intermediate"),
+        ("B2", "Upper-Intermediate"),
+        ("C1", "Advanced"),
+        ("C2", "Proficiency"),
+    )
+    min = MinValueValidator(18)
+    max = MaxValueValidator(80)
+
+    name = forms.CharField(label='name', max_length=100, required=True)
+    sex = forms.ChoiceField(label='sex', choices=SEX_CHOICES, initial='m', widget=forms.RadioSelect, required=True)
+    age = forms.IntegerField(label='age', validators=[min, max], initial=18, required=True)
+    english_level = forms.ChoiceField(label='english level', choices=CEFR)
+
+
+class AuthForm(forms.Form):
+    username = forms.CharField(max_length=100)
+    password = forms.CharField(widget=forms.PasswordInput)
+
+    def clean(self):
+        cleaned_data = super().clean()
+        username = cleaned_data.get('username')
+        password = cleaned_data.get('password')
+        if username and password:
+            self.user = authenticate(username=username, password=password)
+            if self.user is None:
+                raise forms.ValidationError('Please, try again')
+
+
+class RegisterForm(forms.Form):
+    username = forms.CharField(max_length=100, required=True)
+    first_name = forms.CharField(max_length=100, required=True)
+    last_name = forms.CharField(max_length=100)
+    email = forms.EmailField(max_length=100)
+    password = forms.CharField(widget=forms.PasswordInput, required=True)
+    password2 = forms.CharField(widget=forms.PasswordInput, required=True)
+
+    def clean(self):
+        cleaned_data = super().clean()
+        username = cleaned_data.get('username')
+        password = cleaned_data.get('password')
+        password2 = cleaned_data.get('password2')
+
+        if password != password2:
+            raise forms.ValidationError('Passwords are different')
+        try:
+            print(User.objects.get(username=username))
```

Comment:
What is it??

User.objects.filter(username=username).exists()

Try except possible but with DoesNotExist and MultipleObjectsReturned

Reply:
Ooops! No one had to see the print()  ))  
haven't deleted it after debugging
and, again, I know, no one had to use print() for debugging 

Honestly, didn't know about .exists() 
It is a better way than try-except